sky/engine/bindings/jni/jni_array.cc

```diff
@@ -0,0 +1,82 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "sky/engine/bindings/jni/jni_array.h"
+
+#include "sky/engine/bindings/jni/dart_jni.h"
+
+namespace blink {
+
+IMPLEMENT_WRAPPERTYPEINFO(jni, JniArray);
+
+JniArray::JniArray(JNIEnv* env, jarray array)
+    : JniObject(env, array) {}
+
+JniArray::~JniArray() {
+}
+
+jsize JniArray::GetLength() {
+  Dart_Handle exception = nullptr;
+  {
+    ENTER_JNI();
```

Comment:
Super minor nit: The name `ENTER_JNI` makes me look for an `EXIT_JNI`. Can this be called `SCOPED_JNI` or something? I understand if you disregard this :)